### ISSUE TITLE
Is latest code working?

### ISSUE THREAD

**Issue by model-collapse** [vugu/vugu#102]
I am tracking the latest changes on VUGU, it seems the author has made a lot of changes. However, I wrote a simple demo consists of multiple components. However, the referred components in root.vugu only shows an empty html tag in the explorer (such as <foo></foo>). They were not likely executed and rendered.

**Comment by itstmyi** [comment]
Upload your code here please . Or try to download the latest vugu(master branch) and examples for your testing and coding .

**Comment by gabstv** [comment]
@model-collapse  the current components example is outdated.
Now you need to use `<packagename:GoGeneratedComponentName />`. I had to like the render-static_test.go does:
https://github.com/vugu/vugu/blob/c4c2de3e9ddf5cfa914d42e7205ade62b4369e74/staticrender/renderer-static_test.go#L56

**Comment by bradleypeabody** [comment]
Yeah I definitely need to update those docs. Hopefully very soon.
